src/relayserver/RelayServer.ts

```diff
@@ -126,12 +127,12 @@ export class RelayServer extends EventEmitter {
       verifiersToQuery = Array.from(this.trustedVerifiers) as Address[]
     }
 
-    const res: TokenResponse = {}
+    let res: TokenResponse = {}
     for (const verifier of verifiersToQuery) {
       const tokenHandlerInstance = await this.contractInteractor.createTokenHandler(verifier)
-      const acceptedTokens = await tokenHandlerInstance.contract.methods.getAcceptedTokens().call()
-      res[verifier] = acceptedTokens
-    };
+      const acceptedTokens = await tokenHandlerInstance.getAcceptedTokens()
+      res[toChecksumAddress(verifier)] = acceptedTokens
```

Comment:
improvement, and also allows proper testing for the case of empty verifier param